packages/lending/src/protocols/aave-v2/lending-protocol.test.ts

```diff
@@ -620,7 +621,12 @@ describe('Test Aave V2 LendingProtocol', function () {
       it(`${common.toNetworkId(chainId)} market`, async function () {
         const protocol = new LendingProtocol(chainId);
         protocol.setBlockTag(blockTag);
-        const portfolio = await protocol.getPortfolio(account);
+        const _portfolio = await protocol.getPortfolio(account);
+        const portfolio = JSON.parse(JSON.stringify(_portfolio));
+
+        removePortfolioDynamicFields(expected);
+        removePortfolioDynamicFields(portfolio);
```

Comment:
Why do we remove the newly added fields?

Reply:
In the test case, we set the `blockTag` to ensure the portfolio remains consistent within the specified block. However, lstApy is a real-time APY fetched from an external API, meaning the expected and portfolio values might not match.

Reply:
I see. Do we have any test to ensure the lst apy is good to go?